src/geometries/Cube.js

```diff
@@ -138,32 +140,46 @@ CubeTile.prototype.rotY = function() {
 
 
 CubeTile.prototype.centerX = function() {
```

Comment:
I think this can be expressed more succinctly as

```
var levelWidth = this._level.width();
var tileWidth = this._level.tileWidth();
return (this.x * tileWidth + 0.5 * this.width()) / levelWidth - 0.5;
```

which is the same logic as in `FlatView`.